app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -128,7 +129,7 @@ class DraftController @Inject constructor(
             DraftMode.NEW_MAIL -> Unit
         }
 
-        return isSuccess
+        return isSuccess to previousMessage.body!!
```

Comment:
A method called `setPreviousMessage` that returns something ?

Reply:
I extracted the part where we fetch the previous message and therefore could remove the returned values. Good call